api/src/Entity/ContentNode.php

```diff
@@ -40,42 +43,37 @@
     normalizationContext: ['groups' => ['read']],
 )]
 #[ApiFilter(SearchFilter::class, properties: ['parent', 'contentType'])]
+#[Entity(repositoryClass: ContentNodeRepository::class)]
+#[InheritanceType(value: 'JOINED')]
+#[DiscriminatorColumn(name: 'strategy', type: 'string')]
 abstract class ContentNode extends BaseEntity implements BelongsToCampInterface {
-    /**
-     * @ORM\OneToOne(targetEntity="AbstractContentNodeOwner", mappedBy="rootContentNode", cascade={"persist"})
-     */
     #[SerializedName('_owner')]
     #[ApiProperty(readable: false, writable: false)]
+    #[OneToOne(targetEntity: 'AbstractContentNodeOwner', mappedBy: 'rootContentNode', cascade: ['persist'])]
     public ?AbstractContentNodeOwner $owner = null;
 
     /**
      * The content node that is the root of the content node tree. Refers to itself in case this
      * content node is the root.
-     *
-     * @ORM\ManyToOne(targetEntity="ContentNode", inversedBy="rootDescendants")
-     * TODO make not null in the DB using a migration, and get fixtures to run
```

Comment:
Hast du das TODO absichtlich gelöscht?

Reply:
Nö, nicht absichtlich. Rector scheint mit Kommentaren zwischen den Annotations nichts wohlwollend umzugehen...
Will fix.

Reply:
Ok, don't know how to fix...
Comment ist jetzt wieder drin und issue erstellt #2235

Reply:
Ja ich habs auch schon mal debuggt und keine Lösung gefunden... Gut ist das mal ein Issue.